testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveCompatibility.java

```diff
@@ -207,7 +207,7 @@ public void testSmallDecimalFieldWrittenByOptimizedParquetWriterCannotBeReadByHi
 
         // Hive expects `FIXED_LEN_BYTE_ARRAY` for decimal values irrespective of the Parquet specification which allows `INT32`, `INT64` for short precision decimal types
         assertQueryFailure(() -> onHive().executeQuery("SELECT a_decimal FROM " + tableName))
-                .hasMessageMatching(".* org.apache.parquet.io.ParquetDecodingException: Can not read value at 1 in block 0 in file .*");
+                .hasMessageMatching(".* parquet.io.ParquetDecodingException: Can not read value at 1 in block 0 in file .*");
```

Comment:
Maybe we should not validate the package at all? Why the package got changed? Maybe different Hadoop distros are using different parquet libraries?

Reply:
The original PR doesn't have the CI run completed, so I can't tell if it changed, or was just incorrect.

Reply:
Ah this is the approach in https://github.com/trinodb/trino/pull/10847